keyboards/xbows/woody/readme.md

```diff
@@ -0,0 +1,15 @@
+# WOO-DY
+![WOO-DY](https://github.com/moyi4681/qmk_firmware/blob/master/keyboards/xbows/woody/img1.png)
+
+Woo-dy, A 67 Keys Wooden Mechanical Keyboard.
+At once minimalist and stylish, Woo-dy brings a natural calm and relaxation to your working space. The natural wood base made from real walnut or cherry reduces weight, feels great and adds subtle, natural style to any room.
+
+Keyboard Maintainer: X-BOWS  
```

Comment:
X-BOWS does not make this keyboard. The Kickstarter for it is here: https://www.kickstarter.com/projects/woo-dy/woo-dy-a-67-keys-mechanical-keyboard-that-is-breathing/comments

Reply:
I think that this actually needs to be resolved, or the links/wording updated.  

I don't see this keyboard for sale on their site, so listing it as an X-BOWS board seems misleading, if not outright wrong.

Reply:
I'm sure it's a new verson of X-BOWS pcb. don't worry about that.:)

Reply:
It does seem to be in the carousel on the X-BOWS site now. I don't think it was before. Odd.